Tests/Entity/CommentVotable.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the FOSCommentBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace FOS\CommentBundle\Tests\Entity;
+
+use FOS\CommentBundle\Model\VotableCommentInterface;
+
+class CommentVotable extends Comment implements VotableCommentInterface
+{
+    /**
+     * @var int
+     */
+    protected $score = 0;
+
+    /**
+     * @param int $score
+     */
+    public function setScore($score)
+    {
+        $this->score = $score;
+    }
+
+    /**
+     * @return int
+     */
+    public function getScore()
+    {
+        return $this->score;
+    }
+
+    /**
+     * @param int $by
+     */
+    public function incrementScore($by = 1)
+    {
+        $this->score += $by;
+    }
+
+    /**
+     * @param int $by
+     */
+    public function decrementScore($by = 1)
```

Comment:
I think not need, because it is BC break, but when deleting a negative voice (-1) we will need to convert the value to positive to correctly set total score votes.
For example added test https://github.com/FriendsOfSymfony/FOSCommentBundle/pull/667/files#diff-15e87c3ecab320a4304f40a5247c7177R51